source/components/Header/AccountHeader.tsx

```diff
@@ -76,13 +78,60 @@ const RenderAccountsListByBitcoinBased = (
                     )}
                 </li>
               </Tooltip>
-            ))
-        : Object.entries(accounts).map(
-            ([keyringAccountType, accountTypeAccounts]) => (
-              <div key={keyringAccountType}>
-                {Object.values(accountTypeAccounts)
-                  .filter((account) => account.xpub !== '')
-                  .map((account, index, { length }) => (
+            ))}
+
+          {activeNetwork.currency !== 'tsys' &&
```

Comment:
Does trezor works on testnet from evm ?

Reply:
We probably should block it from any utxo testnet that the user might add not only tsys, by checking through the keyringManaget istestnet function that should be straightforward

Reply:
> Does trezor works on testnet from evm ?

Yep, it do.